### DISCUSSION TITLE
GitHub Arctic Code Vault page title typo

### DISCUSSION THREAD

**Discussion by legnoh** [community/community#22687] [New to GitHub]
<p>This page’s title may contain typo. Could you fix it?<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/706834/181072596-b07052e3-7778-4c31-87a6-6bda1056359f.png" class="site-icon" width="72" height="72">
      <a href="https://archiveprogram.github.com/" target="_blank" rel="nofollow noopener">GitHub Archive Program</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/363;"><img src="https://user-images.githubusercontent.com/706834/181072597-14a22f16-453c-47bf-a1c1-02bd29ff9387.png" class="thumbnail" width="690" height="363" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/bb7814fcdd063535c8f4edbc82a5efb00fd0fc04_2_690x363.png, https://aws1.discourse-cdn.com/github/optimized/2X/b/bb7814fcdd063535c8f4edbc82a5efb00fd0fc04_2_1035x544.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/b/bb7814fcdd063535c8f4edbc82a5efb00fd0fc04.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/bb7814fcdd063535c8f4edbc82a5efb00fd0fc04_2_10x10.png"></div>

<h3><a href="https://archiveprogram.github.com/" target="_blank" rel="nofollow noopener">GitHub Archive Program</a></h3>

<p>The GitHub Archive Program will safely story every public GitHub repo for 1,000 years in the Arctic World Archive in Svalbard, Norway.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<blockquote>
<p>GitHub Archive Program | The GitHub Archive Program will safely story every public GitHub repo for 1,000 years in the Arctic World Archive in Svalbard, Norway.</p>
</blockquote>
<ul>
<li>Incorrect: The GitHub Archive Program will safely story …</li>
<li>Correct: The GitHub Archive Program will safely <strong>store</strong> …</li>
</ul>

**Comment by lee-dohm** [comment]
<p>Hello and welcome to the community!</p>
<p>Thanks for reporting this. I’ve submitted a fix for it and am waiting for the owners to approve it.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>The fix has been applied. Thanks again for letting us know :grinning:</p>

**Comment by legnoh** [comment]
<p>LGTM:+1: Thank you for your kindness!</p>
